src/index.html

```diff
@@ -7,8 +7,25 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <img src="./images/logo.png" class="header__logo" alt="moyologo">
+      <nav class="navigation">
+        <ul class="nav__link">
+          <li class="link__point is-active">APPLE</li>
+          <li class="link__point is-active">SAMSUNG</li>
```

Comment:
Each list item should be the link (use a tag with href="#samsung" #apple #laptops and others)